.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contribution Guidelines
+
+## Reporting issues
+
+Please [open an issue](https://github.com/serokell/haskell-algorand-sdk/issues/new/choose) if you find a bug or have a feature request.
+Note: you need to login (e. g. using your GitHub account) first.
+Before submitting a bug report or feature request, check to make sure it hasn't already been submitted
+
+The more detailed your report is, the faster it can be resolved.
+If you report a bug, please provide steps to reproduce this bug and revision of code in which this bug reproduces.
+
+## Code
+
+If you would like to contribute code to fix a bug, add a new feature, or
+otherwise improve our project, pull requests are most welcome.
+
+Our pull request template contains a [checklist](./pull_request_template.md#white_check_mark-checklist-for-your-pull-request) of acceptance criteria for your pull request.
+Please read it before you start contributing and make sure your contributions adhere to this checklist.
+
+## Legal
+
+We want to make sure that our projects come with correct licensing information
+and that this information is machine-readable, thus we are following the
+[REUSE Practices][reuse] – feel free to click the link and read about them,
+but, basically, it all boils down to the following:
+
+  * Add the following header at the very top (but below the shebang, if there
+    is one) of each source file in the repository (yes, each and every source
+    file – it is not as hard as it might sound):
+
+    ```haskell
+    {- SPDX-FileCopyrightText: 2021 Serokell <https://serokell.io>
+     -
+     - SPDX-License-Identifier: MPL-2.0
+     -}
+    ```
+
```

Comment:
Not sure if this is copypasted from somewhere, but it should also recommend using `reuse addheader` which automates the process

Reply:
oh! didn't know that